CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+We welcome contributions by the community. By contributing to gitops-secrets, you agree to abide by the [code of conduct](CODE_OF_CONDUCT.md).
+
+## Code Style
+
+Please check your code prior to submitting pull requests by running:
+
+```sh
+npm run prettier
+npm run lint
+```
+
+## Commit Messages
+
+Commit messages should be verb based, using the following pattern:
+
+- `Fixing ...`
```

Comment:
nit: Could we phrase these in the present simple tense? E.g. `Fixes`, `Adds`, `Updates`, etc.

That will keep this project consistent with our other open-source repos